dopamine/utils/threading_utils.py

```diff
@@ -16,11 +16,23 @@
 
 The `local_attributes` class decorator defines a custom getter, setter and
 deleter for each specified attribute. These getter, setter and deleter actually
-wrap internal attributes that are thread specific.
+wrap internal attributes that are specific to the id of the thread they're
+accessed from.  We call those attributes "thread local".
 
-Each attribute has a callable default value that initializes the local value the
-first time they are called in a thread. To set these default values, use the
-`initialize_local_attributes` helper of this module.
+Each attribute has a callable default value that initializes a thread-local
+variable when it is not bound to a class at the time a caller tries to access
+it.  Typically, this is the first time in a thread that the variable is
```

Comment:
nit: there are too many spaces here.

Reply:
:-) This is a hard habit for me to break when writing documents, too, where the previous author has chosen 1 space between sentences.  I think you identified both places where I'd put 2.  Thanks, and fixed.